lib/wasi/src/wapm/mod.rs

```diff
@@ -503,3 +438,190 @@ fn count_file_system(fs: &dyn FileSystem, path: &Path) -> u64 {
 
     total
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::BTreeMap;
+
+    use super::*;
+
+    const PYTHON: &[u8] = include_bytes!("../../../c-api/examples/assets/python-0.1.0.wasmer");
+    const COREUTILS: &[u8] = include_bytes!("../../../../tests/integration/cli/tests/webc/coreutils-1.0.14-076508e5-e704-463f-b467-f3d9658fc907.webc");
+    const BASH: &[u8] = include_bytes!("../../../../tests/integration/cli/tests/webc/bash-1.0.12-0103d733-1afb-4a56-b0ef-0e124139e996.webc");
+
+    #[test]
```

Comment:
We should also have a test that makes sure the current static webserver webcs continue to work with the changes.

(they should, but wouldn't want to break this)

They currently rely on commands from a dependency being automatically used, like:

```
[package]
name = 'christoph/tmp'
version = '0.1.0'
description = 'christoph/tmp website'

[dependencies]
"sharrattj/static-web-server" = '1'

[fs]
public = 'public'
```

Reply:
I've added a test based on https://wapm.dev/wasmer/hello and it passes. Can you have a look and see if I need to make more assertions?